packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -0,0 +1,322 @@
+
+1. In the Admin Console, go to **Security** > **API**.
+2. On the **Authorization Servers** tab, select the edit icon to the right of the authorization server that you want to use. In this use case example, we are using the `default` custom authorization server.
+3. Select **Access Policies**.
+4. Select the edit icon for the Default Policy Rule.
+5. Select **Client-initiated backchannel authentication (CIBA)** as a grant type and click **Update Rule**.
+
+## Enroll a custom authenticator to use CIBA
+
+Use the Devices SDK and your app to enroll a custom authenticator for the test user. In this use case example, we are using the Okta Authenticator Sample App and the xCode simulator to enroll the authenticator. See the **What you need** section if you haven’t set up the sample app.
+
+1. In the **PushSampleApp** directory of the Okta Devices SDK, run the Okta Authenticator Sample App and then open the mobile app in the simulator.
+2. Click **Sign In**, enter the test user’s credentials, and click **Sign In**.
+3. Complete any additional two-factor authentication required and click **Verify**.
+4. On the Sign in securely using push notifications screen, click **Set up**.
+5. On the Security Settings screen, slide to enable **Sign in with push notifications**.
+6. Click **Ok** at the confirmation dialog. Additional security setting options then appear, including **Enable CIBA transactions**.
+7. Slide to select **Enable CIBA transactions**.<br>
+   The sample app is set up to include the **Enable CIBA transactions** option by default for the user to enable CIBA themselves in the mobile app. However, you can implement CIBA in your app any way that you want, for example, create an enrollment flow that turns CIBA on by default, making it transparent to users.
+8. Click **Ok** at the success dialog and close the Security Settings screen.
+9. Leave your test user signed in to the authentication device (the Magenta Bank app on the xCode simulator).
+  <!-- **Note:** See the [MyAccount API](/docs/reference/api/myaccount/) for examples of [enrolling](?link to new api operation?) and [updating](?link to new api operation?) a custom app authenticator to use CIBA by adding `CIBA` to the `transactionTypes` array. -->
+
+## Test the CIBA flow
+
+This section walks you through how to test the CIBA authentication flow using the Okta Authenticator Sample App and the custom authenticator that you previously set up.
+
+### Base64-encode the client ID and secret
+
+The call to the `/bc/authorize` endpoint and to the `/token` endpoint requires authentication. In this case, it’s a Basic Auth digest of the client ID and secret of the OIDC client that you created in [an earlier section of the guide](#create-an-oidc-client-integration-and-enable-CIBA). You can find them on the client’s **General** tab in the **Client Credentials** section.
```

Comment:
Check this link/anchor. It wasn't working in my preview. #create-an-oidc-client-integration-and-enable-ciba is the link in my preview (lower-case ciba)